### PR TITLE
Fal

### PR DESCRIPTION
**Pull request by stevekrouse**

### CODE REVIEW

**Review by drochetti — APPROVED** [review]
Ok, better late than never. I love the blog post, it's concise, and the result is mind-blowing. The Townie part is very cool, I wanna try it myself.

Ship it!

File: src/content/blog/fal.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Val Town and Fal Partnership
+description: Bringing lightning fast AI image generation to Val Town
+pubDate: Oct 21, 2024
+author: Steve Krouse
+---
+
+import { Image } from "astro:assets";
+
+{/* https://supabase.com/blog/supabase-vercel-partnership */}
```
Comment (drochetti) [inline src/content/blog/fal.mdx:10]: leftover I guess?

File: src/content/blog/fal.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    /* Return HTML */
+  }
+
+  if (url.pathname === "/api/fal/proxy") {
+    return falProxyRequest(req);
+  }
+}
+```
+````
+
```
Comment (drochetti) [inline src/content/blog/fal.mdx:79]: small suggestion: we could briefly talk about the Flux Schnell, the fast image model being used here and link `https://fal.ai/flux` for more info.

**Review by drochetti — APPROVED** [review]

File: src/content/blog/fal.mdx (hunk 2 above)
Comment (stevekrouse) [reply src/content/blog/fal.mdx:79]: done!

**Review by tmcw — APPROVED** [review]

**Pull request merged**